### PR TITLE
Scaling Background Art Added

### PR DESCRIPTION
**Pull request by IsaacS2**
I added a few unused scripts in the Scripts folder that contain some implementations for scrolling backgrounds and scaling backgrounds; however, if they don't look like they'll be useful in the future, I can just delete them.

### CODE REVIEW

File: Assets/Sprites/Background Materials.meta
Comment (adenprince) [inline Assets/Sprites/Background Materials.meta:1]: I think the material files can be removed. We're not using a Mesh Renderer for the background, so we don't need to specify a material for the background.

File: Assets/Scripts/Background/SpriteStretch.cs [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+        //
+        spriteWidth = spriteSize.x;
+        scaleFactorX = worldSpaceWidth / spriteSize.x;
+        gameObject.transform.localScale = new Vector3(scaleFactorX, height, 1);
+    }
+
+    // Update is called once per frame
+    void Update()
+    {
+        newSize = Camera.main.ScreenToWorldPoint(new Vector3(Screen.width, Screen.height, Camera.main.transform.position.z));
```
Comment (adenprince) [inline Assets/Scripts/Background/SpriteStretch.cs]: This code is taking a point in screen coordinates (in pixels) and converting it to world coordinates (in world units that are used for GameObject positions) using the main camera and then getting the screen size in world coordinates using that point and the camera position.
It's simpler to use `Camera.main.orthographicSize` to get half of the camera view height in world coordinates. Then, you can multiply the camera view height by `Camera.main.aspect` to get the width. This is similar to what's in `FullScreenSprite.cs`. That script needs to be modified to not set the position to 0, run in the Update method on screen size change, and keep the height constant by only changing the x scale.

File: Assets/Scripts/Background/ScaleBackgroundOld.cs [hunk 2]
```diff
@@ -0,0 +1,44 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+//
+// Old script for scaling the background (no longer used)
```
Comment (adenprince) [inline Assets/Scripts/Background/ScaleBackgroundOld.cs]: I think the unused scripts can be removed.

File: ProjectSettings/TagManager.asset [hunk 3]
```diff
@@ -38,6 +38,12 @@ TagManager:
   - 
   - 
   m_SortingLayers:
+  - name: Background
+    uniqueID: 847899343
+    locked: 0
   - name: Default
     uniqueID: 0
     locked: 0
+  - name: Player
```
Comment (adenprince) [inline ProjectSettings/TagManager.asset]: It seems like the added sorting layers are unused.

**Review by adenprince — CHANGES_REQUESTED** [review]
Looks good. I think some unused files should be removed and the background scaling code should be changed.

File: ProjectSettings/TagManager.asset (hunk 3 above)
Comment (IsaacS2) [reply ProjectSettings/TagManager.asset]: Gotcha; I deleted the "Player" layer and used the "Background" layer for the background.

File: Assets/Sprites/Background Materials.meta
Comment (IsaacS2) [reply Assets/Sprites/Background Materials.meta:1]: Gotcha; I deleted that file now.

File: Assets/Scripts/Background/ScaleBackgroundOld.cs (hunk 2 above)
Comment (IsaacS2) [reply Assets/Scripts/Background/ScaleBackgroundOld.cs]: Gotcha; I deleted those now.

File: Assets/Scripts/Background/SpriteStretch.cs (hunk 1 above)
Comment (IsaacS2) [reply Assets/Scripts/Background/SpriteStretch.cs]: Gotcha; I fixed up the scaling script.

**Review by adenprince — APPROVED** [review]

**Pull request merged**